pennylane/gradients/pulse_gradient.py

```diff
@@ -576,26 +626,29 @@ def ansatz(params):
     return _expval_stoch_pulse_grad(tape, argnum, num_split_times, key, shots, use_broadcasting)
 
 
-def _generate_tapes_and_cjacs(tape, idx, key, num_split_times, use_broadcasting):
+def _generate_tapes_and_cjacs(tape, op_id, key, num_split_times, use_broadcasting, par_idx=None):
```

Comment:
Maybe operation would be better as op_id

Reply:
Done.